lms/djangoapps/experiments/tests/test_utils.py

```diff
@@ -116,3 +121,19 @@ def test_price_and_sku_from_multiple_courses(self):
         self.assertEqual(2, len(skus))
         self.assertIn(self.run_a_sku, skus)
         self.assertIn(self.entitlement_a_sku, skus)
+
+    def test_get_experiment_user_metadata_context(self):
+        user_metadata_expected_result = """{'username': 'robot0', 'user_id': 2, 'course_id': 'org.0/course_0/Run_0',
```

Comment:
does the method return a string rather than a dictionary?  i imagine it should probably return a dictionary
either way maybe you could put this as a dictionary and have each key on a separate line to make this a bit more readable

Reply:
Sorry for the false alarm- the method still returns a dictionary, I'd incorrectly had quotes in my test but removed them now